editor/project_converter_3_to_4.cpp

```diff
@@ -374,6 +374,7 @@ static const char *gdscript_function_renames[][2] = {
 	{ "get_space_override_mode", "get_gravity_space_override_mode" }, // Area2D
 	{ "get_spatial_node", "get_node_3d" }, // EditorNode3DGizmo
 	{ "get_speed", "get_velocity" }, // InputEventMouseMotion
+	{ "get_stylebox", "get_theme_stylebox" }, // Theme
```

Comment:
That's wrong for two reasons. First of all, in `Theme` this methods wasn't renamed, it was renamed in `Control`. So the comment is incorrect here. Second, because this method also exists in `Theme`, and wasn't renamed there, this will result in false positives. Of course, code related to Control nodes is more likely to be the case in an arbitrary code base, but that's still concerning.

Also, there are other methods, `get_color`, `get_font`, `get_constant`, `get_icon`, as well as `has_*` equivalents. So it would make sense to either add all of them, or indeed nothing to avoid false positives. Note that `get_icon` also conflicts with the same method of `Button` (it's a bug in 3.x, actually), so even if we agree to add the other rules, icon-related methods may still be better left untouched.

Reply:
Ahh I see. Never actually instantiated the `Theme` class myself. Thanks for the hint!
Then `get_stylebox` is probably a bad candidate. I will remove it and check out the other theme methods. :)

Reply:
Ah no, those are all Control methods. So they all would be bad candidates then, for the same reason (overlapping with Theme's methods).